src/modularityDummy/java/org/spongepowered/include/com/google/common/base/PatternCompiler.java

```diff
@@ -0,0 +1,5 @@
+package org.spongepowered.include.com.google.common.base;
+
+// Dummy public class so that the module can declare a uses directive for it
+public interface PatternCompiler {
```

Comment:
This feels terrible, are you sure it works. After adding the license header to this file to make sure it builds, looking into the built jar I see the empty interface, not the orginal.

Reply:
I see an empty interface too, but its access modifiers are `abstract interface`, not `public abstract interface` as the one we add. The interface is as empty in the `0.12.5+mixin.0.8.5` version (latest) currently published on maven.

Reply:
Thanks for checking. Still not happy with it, but if it works then sure.